backend/app/ops/masking.py

```diff
@@ -63,42 +63,43 @@ async def call(self, input: str) -> tuple[str, Mapping[str, str]]:
         analyzer = Analyzer()
         anonymizer = Anonymizer()
         analysis = await analyzer.analyze(AnalyzerRequest(text=input))
-        anonymized = await anonymizer.anonymize(
-            AnonymizerRequest(
-                text=input,
-                anonymizers=Anonymizers(
-                    PERSON=Keep(),
-                    PHONE_NUMBER=Keep(),
-                    LOCATION=Keep(),
-                    CREDIT_CARD=Keep(),
-                    EMAIL_ADDRESS=Keep(),
-                    IBAN_CODE=Keep(),
-                    DEFAULT=Replace(),
+        anonymized = await anonymizer.anonymize(AnonymizerRequest(
+            text=input,
+            anonymizers=Anonymizers(
+                PERSON=Keep(),
+                LOCATION=Keep(),
+                EMAIL_ADDRESS=Keep(),
+                DEFAULT=Keep(),                                    
                 ),
                 analyzer_results=analysis,
             )
         )
         mapping = {}
+        print("anonymized",anonymized)
+        for item in anonymized.items:
+            if item.entity_type in ["PERSON", "LOCATION", "EMAIL_ADDRESS"]:
+                print(f"{item.entity_type}: {item.text}")
         for item in anonymized.items:
             # Compute a replacement value
             if item.entity_type == "PERSON":
                 replacement = self.replace_person(item.text, input)
-            elif item.entity_type == "PHONE_NUMBER":
-                replacement = self.replace_person(item.text, input)
+                print("PERSON", replacement, item.text)
+            #elif item.entity_type == "PHONE_NUMBER":
```

Comment:
you can remove rather than comment each line and add a general comment explaining the approach